js/src/Figure.ts

```diff
@@ -274,6 +271,12 @@ class Figure extends widgets.DOMWidgetView {
         });
     }
 
+    debouncedRelayout() {
+        _.debounce(() => {
```

Comment:
Not sure that works. `_.debounce` returns the debounced function. So `debouncedRelayout` must be the output of `_.debounce`, it should not call it.

Reply:
I saw this issue multiple times, and I am not sure how to properly fix it in TypeScript. But the proper JavaScript way of doing it might be to change the prototype manually.

Reply:
oh right